src/ui/common/src/components/CodeBlock/index.tsx

```diff
@@ -8,6 +8,7 @@ type Props = {
 };
 
 export const CodeBlock: React.FC<Props> = ({ language, children }) => {
+  console.log('codeblock children: ', children);
```

Comment:
log

Reply:
good catch, should be removed and things should be linted as well.